packages/cerebral/typings/__test__/examples/code1.ts

```diff
@@ -0,0 +1,12 @@
+// if i try importing from 'cerebral' and 'cerebral/devtools' , it fails to compile
```

Comment:
let's remove this comment.
it's expected behavior. js tests also requires using relative pathes.
btw, why this file burried so deep? can we collocate tests near d.ts files?